src/lib/connectors/reporting/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const config = require('../../../../config');
+const urlJoin = require('url-join');
+const got = require('got');
+
+/**
+ * Requests a report from the Reporting microservice
+ * @param request {Object} The request object
+ * @returns {Request} Got Request - Streams file from S3
+ */
+const getReport = request => {
```

Comment:
As with the UI one, I think this function should just take the needed arguments something like
```
getReport(key, userId) {
...
```

Reply:
Done in 3ee7c6cb